dnf-behave-tests/features/makecache.feature

```diff
@@ -0,0 +1,13 @@
+Feature: makecache command 
+
+
+@xfail
+@fixture.httpd
+@1745170
+Scenario: disabled makecache --timer does not invalidate cached metadata
+Given I use the http repository based on "dnf-ci-fedora"
+  And I successfully execute dnf with args "makecache"
+ When I execute dnf with args "makecache --timer --setopt=metadata_timer_sync=0"
+  And I execute dnf with args "install setup"
+ # This line matches downloading of metadata (generally first line in transaction output)
+ Then stdout does not contain "http-dnf-ci-fedora\s+.+B\/s \| .+B"
```

Comment:
I understand you want to check for the sync line here, as that is what you care about and the rest of the output is irrelevant...

So just a note perhaps (Feel free to leave it as it is), I personally find the regexes hard to read and therefore annoying. While this is a simple case, we do have much longer ones. Sometimes they don't match and you may need to spend a long time trying to figure out what exactly it does and what causes the problem. So the point is, I guess, to try to avoid them and only use them if necessary.

Reply:
Yes its probably my fault I have been using the regexes a lot to make my life easier, maybe at the cost of readability. I will try to do better.

Reply:
Especially in this negative form - stdout does not contain "regex" I consider it fragile. If dnf change its output format, the test will continue to pass regardless dnf downloads the metadata or not. The better approach will probably be "positive" testing stdout is "the output".

Reply:
That's actually a very good point. I agree on the `stdout is ...` front :slightly_smiling_face: